src/pico/pico-screens/screens/shared.h

```diff
@@ -0,0 +1,40 @@
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef __SCREEN_SHARED__
+#define __SCREEN_SHARED__
+
+#include "pico.h"
+#include "i_video.h"
+
+
+#define DOOM_WIDTH SCREENWIDTH
+#define DOOM_HEIGHT SCREENHEIGHT
+
+// here be some chicanery to avoid floats
+#ifndef DOWNSAMPLING_FACTOR_OUT_OF_100
+#define DOWNSAMPLING_FACTOR_OUT_OF_100 243
+#endif
+
+#define DOWNSAMPLED_WIDTH ((SCREENWIDTH * 100 / DOWNSAMPLING_FACTOR_OUT_OF_100))
```

Comment:
this is currently all that drives the width / height of the image. we could flip the script and divine the downsampling factor from the size of the screen, I suppose...